content/id/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Disrupsi
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+Petunjuk ini diperuntukkan bagi pemilik aplikasi yang ingin membuat aplikasi yang memiliki ketersediaan yang tinggi, sehingga dibutuhkan untuk mengerti jenis-jenis Gangguan yang dapat terjadi pada Pods.
+
+Petunjuk ini juga diperuntukkan bagi administrator kluster yang ingin melakukan berbagai tindakan otomasi pada kluster, seperti pembaruan dan *autoscaling* kluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Disrupsi yang Disengaja dan Tidak Disengaja
+
+Pods tidak akan terhapus hingga sesuatu (orang maupun *controller*) menghancurkan mereka atau ada kesalahan perangkat keras maupun perangkat lunak yang tidak dapat dihindari.
+
+Kita menyebut kasus-kasus yang tidak dapat dihindari sebagai *disrupsi yang tidak disengaja* terhadap aplikasi. Beberapa contohnya adalah sebagai berikut:
+
+- Kesalahahan perangkat keras pada mesin yang menjalankan Node
+- Administrator kluster menghapus *virtual machine* secara tidak sengaja
+- Kesalahan pada penyedia layanan cloud yang mengakibatkan terhapusnya *virtual machine*
+- Sebuah *kernel panic*
+- Node menghilang dari kluster karena partisi jaringan kluster
+- Diusirnya Pod karena Node [kehabisan sumber daya](/docs/tasks/administer-cluster/out-of-resource)
+
+Dengan pengecualian pada kondisi kehabisan sumber daya, kondisi-kondisi tersebut pada umumnya diketahui oleh kebanyakan pengguna; Kondisi-kondisi tersebut tidak hanya terjadi pada Kubernetes
+
+Kita menyebut kasus-kasus lainnya sebagai *disrupsi yang disengaja*. Hal ini termasuk tindakan yang dilakukan oleh pemilik aplikasi atau yang dilakukan oleh administrator kluster. Pemilik aplikasi umumnya melakukan hal-hal berikut:
+
+- Menghapus Deployment atau pengontrol yang mengatur Pod
+- Memperbarui templat Pod yang menyebabkan pengulangan kembali/*restart*
+- Menghapus Pod secara langsung
+
+Administrator kluster umumnya melakukan hal-hal berikut:
+
+- [Menguras Node](/docs/tasks/administer-cluster/safely-drain-node/) untuk perbaikan atau pembaruan.
```

Comment:
nit: _Draining_ node idk but the context seems a little bit different between "menguras" in kbbi and _draining_ here

Reply:
What I understand from _drain_ in the kubectl command is that we want to evict/clean all pods from a given node safely (respecting the PDB) to prepare the node for upgrades.
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#drain

I think it has the same meaning in Indonesian, we will say "menguras node" to evict/clean all pods from a given node.
.
.
Or, considering the `kubectl drain` command is using _drain_, we can also italicize all _drain_ and _draining_ in the docs.
This approach could be better if we want to familiarize future readers with the `drain` command and its meaning (by relating it with its usage).
.
.
What do you think?

Reply:
> Or, considering the kubectl drain command is using drain, we can also italicize all drain and draining in the docs.
This approach could be better if we want to familiarize future readers with the drain command and its meaning (by relating it with its usage).

I think this one make more sense since it's mostly related to the `drain` command in `kubectl`, probably: melakukan *drain* for most cases